src/frontend/src/components/Booking/BookingCard.tsx

```diff
@@ -0,0 +1,216 @@
+"use client";
+import photographerGalleriesService from "@/services/photographerGalleries";
+import userService from "@/services/user";
+import { Booking, BookingStatus } from "@/types/booking";
+import Image from "next/image";
+import { useEffect, useState } from "react";
+import { User } from "@/types/user";
+import { UserResponse } from "@/types/response";
+import { GalleryResponse } from "@/types/response";
+import { isDifferentDay, transformDate } from "@/utils/date";
+
+
+
+interface BookOptions {
+  props: Booking;
+  openModal: Function;
+  setModalProps: Function;
+}
+
+interface customerProp {
+  profile_picture_url: string | undefined;
+  data: {
+    firstname: string | undefined;
+    lastname: string | undefined;
+    username: string | undefined;
+  };
+}
+
+interface galleryProp {
+  location: string | undefined;
+  price: number | undefined;
+  galleryname: string | undefined;
+}
+export default function BookingCard(options: BookOptions) {
+  const [customer, setCustomer] = useState<customerProp | null>();
+  const [gallery, setGallery] = useState<galleryProp | null>();
```

Comment:
From what I've read, this gallery is declared but never used, right? If so, please consider removing it for cleaner code.